plugins/zynaddsubfx/CMakeLists.txt

```diff
@@ -2,11 +2,13 @@ INCLUDE(BuildPlugin)
 
 
 # definitions for ZynAddSubFX
-IF(LMMS_BUILD_LINUX OR LMMS_BUILD_APPLE)
+IF(LMMS_BUILD_LINUX OR LMMS_BUILD_APPLE OR LMMS_BUILD_OPENBSD)
+    FIND_PACKAGE(X11)
+    INCLUDE_DIRECTORIES(${X11_INCLUDE_DIR})
 	ADD_DEFINITIONS(-DOS_LINUX)
-ELSE(LMMS_BUILD_LINUX OR LMMS_BUILD_APPLE)
+ELSE(LMMS_BUILD_LINUX OR LMMS_BUILD_APPLE OR LMMS_BUILD_OPENBSD)
```

Comment:
> Is it correct with older version of cmake ?

Yes, this is a readability practice.  e.g. if you have many nested `IF()`'s.


```cmake 
IF(FOO)
  MESSAGE("IN FOO")
  IF(BAR)
    MESSAGE("IN BAR")
  ENDIF(FOO)
ENDIF(BAR)
```

It's fine to leave, but we've been slowly cleaning them up in many places as we visit them as they start to loose their effectiveness when there are too many params.

```cmake 
IF(FOO OR BAR OR LOREM OR IPSUM OR DOLOR OR SIT OR AMET)
  MESSAGE("HELLO FOO BAR LOREM IPSUM DOLOR SIT AMET")
ENDIF(FOO OR BAR OR LOREM OR IPSUM OR DOLOR OR SIT OR AMET)
```